### ISSUE TITLE
Missing space detection for invalid inline comments

### ISSUE THREAD

**Issue by nait-aul** [PyCQA/pylint#1124]
### Steps to reproduce

```python
if 1 > 0: # NOTE: There should be 2 spaces between : and #, only 1 is.
    pass
```

### Current behavior

```bash
Your code has been rated at 10.00/10 ...
```

### Expected behavior

```
E: ... : at least two spaces before inline comment
```

PEP 8:

> Inline comments should be separated by at least two spaces from the statement. They should start with a # and a single space.

### pylint --version output

```bash
1.6.4
```

**Comment by PCManticore** [comment]
I am closing this. Pylint contains only a subset of PEP-8 checks and it will probably not grow any more of them. Feel free to use it in combination with the pycodestyle tool, which does a better job at enforcing PEP-8 compliance than Pylint ever will.

**Comment by y2kbugger** [comment]
If this ever gets added in an extension, I would hope that this would also include a check for making sure that there is at least one space between `#` and the comment start.
